src/Accord.Bot/CommandGroups/UserChannelBlockingCommandGroup.cs

```diff
@@ -0,0 +1,221 @@
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Accord.Bot.Helpers;
+using Accord.Services.Helpers;
+using Accord.Services.UserChannelBlocks;
+using MediatR;
+using Remora.Commands.Attributes;
+using Remora.Commands.Groups;
+using Remora.Discord.API.Abstractions.Objects;
+using Remora.Discord.API.Abstractions.Rest;
+using Remora.Discord.API.Objects;
+using Remora.Discord.Commands.Conditions;
+using Remora.Discord.Commands.Contexts;
+using Remora.Discord.Commands.Parsers;
+using Remora.Results;
+
+namespace Accord.Bot.CommandGroups
+{
+    [Group("channel")]
+    public class UserChannelBlockingCommandGroup : CommandGroup
+    {
+        private readonly ICommandContext _commandContext;
+        private readonly IMediator _mediator;
+        private readonly IDiscordRestChannelAPI _channelApi;
+        private readonly IDiscordRestGuildAPI _guildApi;
+        private readonly CommandResponder _commandResponder;
+        private readonly DiscordCache _discordCache;
+        private readonly DiscordAvatarHelper _discordAvatarHelper;
+
+        public UserChannelBlockingCommandGroup(ICommandContext commandContext,
+            IMediator mediator,
+            IDiscordRestChannelAPI channelApi,
+            IDiscordRestGuildAPI guildApi,
+            CommandResponder commandResponder,
+            DiscordCache discordCache,
+            DiscordAvatarHelper discordAvatarHelper)
+        {
+            _commandContext = commandContext;
+            _mediator = mediator;
+            _channelApi = channelApi;
+            _guildApi = guildApi;
+            _commandResponder = commandResponder;
+            _discordCache = discordCache;
+            _discordAvatarHelper = discordAvatarHelper;
+        }
+
+        [RequireContext(ChannelContext.Guild), Command("hidden"), Description("Display your hidden channels")]
+        public async Task<IResult> GetHiddenChannels()
+        {
+            var usersBlockedChannels = await _mediator.Send(new GetUserBlockedChannelsRequest(_commandContext.User.ID.Value));
+
+            if (usersBlockedChannels.Count == 0)
+            {
+                await _commandResponder.Respond("You don't have any hidden channel yet");
+            }
+            else
+            {
+                StringBuilder sb = new();
+                foreach (var blockedChannel in usersBlockedChannels)
+                {
+                    sb.AppendLine($"{DiscordFormatter.ChannelIdToMention(blockedChannel)}");
+                }
+
+                await _commandResponder.Respond(new Embed
+                {
+                    Author = new EmbedAuthor(
+                        DiscordHandleHelper.BuildHandle(_commandContext.User.Username, _commandContext.User.Discriminator),
+                        IconUrl: _discordAvatarHelper.GetAvatarUrl(_commandContext.User)),
+                    Title = "Hidden Channels",
+                    Description = sb.ToString()
+                });
```

Comment:
This likely needs an issue creating, as in future this could cause Discord to fail sending the embed if a user has too many hidden channels - right now it's not a concern of mine, because I doubt people will have _that_ many channels hidden.

Reply:
would you like to create a "paging" as i have in reminders?